climada/hazard/trop_cyclone.py

```diff
@@ -410,12 +438,10 @@ def compute_windfields(track, centroids, model):
     if npositions < 2:
         return windfields, reachable_centr_idx
 
-    # never use longitudes at -180 degrees or below
-    t_lon[t_lon <= -180] += 360
-
-    # only use longitudes above 180, if 180 degree border is crossed
-    if t_lon.min() > 180:
-        t_lon -= 360
+    # normalize longitude values
+    mid_lon = 0.5 * sum(lon_bounds(t_lon))
+    lon_normalize(t_lon, center=mid_lon)
+    lon_normalize(centroids[:,1], center=mid_lon)
```

Comment:
The normalization of longitudinal values around some "central" longitude (see https://en.wikipedia.org/wiki/Sinusoidal_projection) is very important for the sinusoidal distance computations. It was not necessary at all for the spherical geodesic distances. However, the loss in performance is really negligible according to some performance tests I did.

Reply:
This is now mentioned in the comment above and in the docstring of `_close_centroids`.